src/stemmablenote.ts

```diff
@@ -141,15 +163,15 @@ export class StemmableNote extends Note {
     if (this.flag) {
       this.buildFlag();
     }
```

Comment:
@0xfe I would say that flag is never initialised. Is this a bug?

Reply:
I have checked that this is not a bug, `buildFlag` is called in `StaveNote` however `this.flag` could be replaced by `this.hasFlag()` for consistency with the other checks, your choice.

Reply:
Modified to use consitently the same check.